max.rb

```diff
@@ -0,0 +1,80 @@
+# Method name: max
+# Input: a list of numbers
+# Returns: the largest number in the list
+# Prints: Nothing
+#
+# In English, the "max" method takes as input a list of numbers and
+# returns the largest number in that list.
+
+# Note: This exercise is "complete."
+# This is meant to give you a starting point and also illustrate what we're looking for.
+# There is ***NO NEED*** to comment line-by-line, although it can't hurt.
+# It's not common in everyday, professional Ruby, but here we're in a learning environment.
+=begin
+def max(list)
+  max_so_far = list.first  # or, equivalently, list[0]
+  list.each do |item|      # or, equivalently, for item in list
+    if item > max_so_far   # if current item is greater than max so far
+      max_so_far = item    #   set max so far to current item
+    end
+  end
+
+  return max_so_far        # we've gone through entire list, return max so far
+end
+=end
+def max(list)
+  max_so_far = list.first
+  list.each do |item|
+    if item > max_so_far
+      max_so_far = item
+    end
+  end
+  max_so_far
+end
+
+
+# This "if" statement will be gobblededook, but it means:
```

Comment:
This feedback is just for the sake of thoroughness.  In a case like this, where these comments were originally added for pedagogical reasons, I'd 100% remove them before committing.